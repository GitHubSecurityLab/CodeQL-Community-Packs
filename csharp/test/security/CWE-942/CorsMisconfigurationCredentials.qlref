security/CWE-942/CorsMisconfigurationCredentials.ql