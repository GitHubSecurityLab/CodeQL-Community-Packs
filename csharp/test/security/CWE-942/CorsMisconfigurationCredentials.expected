| CorsMiconfigurationCredentials.cs:12:7:15:10 | call to method AddPolicy | The following CORS policy may allow credentialed requests from 3rd party websites |
